pay-api/tests/conftest.py

```diff
@@ -80,7 +80,7 @@ def db(app):  # pylint: disable=redefined-outer-name, invalid-name
         return _db
 
 
-@pytest.fixture(scope='function', autouse=True)
+@pytest.fixture(scope='function')
```

Comment:
Move this to optional, Ody expressed some concern when unit testing versioning etc.